fetch.bs

```diff
@@ -6424,6 +6428,11 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
    <a lt="obtain a WebSocket connection">obtaining a WebSocket connection</a>, given
    <var>request</var>'s <a for=request>current URL</a>.
 
+   <dt>"<code>webtransport</code>"
+   <dd><p>Let <var>connection</var> be the result of
+   obtaining a WebTransport connection, given
```

Comment:
Link to [obtain a WebTransport connection](https://pr-preview.s3.amazonaws.com/jan-ivar/webtransport/pull/697.html#obtain-a-webtransport-connection) blocked on https://github.com/w3c/webtransport/pull/697.
```suggestion
   <a lt="obtain a WebTransport connection">obtaining a WebTransport connection</a>, given
```